pkg/api/v1/persistentvolume/util.go

```diff
@@ -147,6 +147,11 @@ func VisitPVSecretNames(pv *corev1.PersistentVolume, visitor Visitor) bool {
 				return false
 			}
 		}
+		if source.CSI.NodeExpandSecretRef != nil {
```

Comment:
Don't we need something in `DropDisabledFields` function of pv type, so as if feature is disabled and users try to set it, it gets dropped?

Reply:
No needed,  if the feature-gates is disabled, we don't pass it to external-provisioner.

Reply:
Not sure I understand. Regardless of what we do in external-provisioner, we need to drop the field if feature gate is not enabled. We carried similar logic for every other field while feature was in alpha.

Reply:
See examples of other alpha fields being handled - https://github.com/kubernetes/kubernetes/commit/2c1b74376691cacf7b56904fe696df45753a8f99